Updated instructions based on feedback portal (#51)
telerik/justmock-docs: getting-started/using-telerik-justmock-in-your-test-project.md

	**Figure 1: Add new project**  
	![Add New Project to VS Solution](images/AddNewProject.png)

> 1. Select the Test Project template or search for it by typing "JustMock test". After that select the Test Project template and click OK.

Reviewer: ```suggestion
1. Select the desired **JustMock Test Project** template from the menu and click OK. 
```
The first sentence instructs the reader to select the Test Project template. The second sentence duplicates the same instructions and asks the reader to  select the Test Project template again.
We can omit the template search instruction, I am not sure if the add more value.

	**Figure 2: Choose the JustMock Test Project template**  

> 	![JustMock Test Project Template](images/ProjectTemplate1.png)

Reviewer 2: Please replace the image or remove the old and add a new one with a different name if required. And please try to name the images according to their usage.
